news_items/002_in_browser_editing/index.html

```diff
@@ -0,0 +1,39 @@
+
+<p>
+  Thanks to a huge amount of work
+  by <a href="https://github.com/tbretl">Tim Bretl</a>, PrairieLearn
+  now allows you to directly edit all course content directly within
```

Comment:
Maybe change "to directly edit" -> "to edit" (too many directly's)

Reply:
done